modules/DesktopDataLaboratory/src/main/java/org/gephi/desktop/datalab/tables/EdgesDataTable.java

```diff
@@ -92,8 +92,19 @@ public Class getColumnClass() {
 
             @Override
             public Object getValueFor(Edge edge) {
-                return edge.getTypeLabel() != null ? edge.getTypeLabel().toString() : null;
+                return edge.getTypeLabel() != null ? edge.getTypeLabel().toString() : "";

```

Comment:
Is this necessary? Let's test the effect of editing the row and setting the empty string again. Does it change the edge type to a new one?

Maybe in `setValueFor` we should check for empty string and in that case `setType(null)`. And even safer, check for empty string after trimming, to not generate weird blank string edge types on accident.

Apart from this, LGTM! 👍🏼

Reply:
Yeap right, I've changed the behaviour to have empty string that mapped to kind type 0

Reply:
Thanks @totetmatt!